tests/src/test/scala/ch/renku/acceptancetests/MinimalFunctionalitySpec.scala

```diff
@@ -34,6 +36,16 @@ class MinimalFunctionalitySpec
     removeProjectInGitLab
     verifyProjectWasRemoved
 
+    launchUnprivilegedEnvironment(anonEnvConfig)
+    stopEnvironment(anonEnvConfig.projectId)
+
     logOutOfRenku
+
+    launchAnonymousEnvironment(anonEnvConfig) match {
```

Comment:
How do you feel doing that this way:
`launchAnonymousEnvironment(anonEnvConfig) map (_ => stopEnvironment(anonEnvConfig.projectId))`

Reply:
You are right, that looks like more idiomatic scala.